sdcm/cluster.py

```diff
@@ -345,6 +345,7 @@ def __init__(self, name, ssh_login_info=None, base_logdir=None, node_prefix=None
                                        DatabaseLogEvent(type='SEMAPHORE_TIME_OUT', regex='semaphore_timed_out'),
                                        DatabaseLogEvent(type='BOOT', regex='Starting Scylla Server', severity=Severity.NORMAL)]
 
+        self._database_error_patterns = ['std::bad_alloc', 'integrity check failed', 'Failed to load schema version']
```

Comment:
@amoskong 
you need to add a new `DatabaseLogEvent` to `self._database_error_events`, `self._database_error_patterns` isn't being used anymore

something like this:
```python
DatabaseLogEvent(type='SCHEMA_FAILURE', regex='Failed to load schema version')
```

Reply:
Fixed.